packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2417,6 +2417,9 @@ class _ElementLocationStatsTracker {
     }
     final _HasCreationLocation creationLocationSource = widget;
     final _Location location = creationLocationSource._location;
+    if(location == null) {
```

Comment:
Something has gone wrong with the kernel transformer to track widget creation locations. If you have a repro for the underlying problem it would be good to fix it rather than papering it over here.